filament/src/materials/skybox.mat

```diff
@@ -57,8 +65,8 @@ fragment {
     }
 
     float3 shaprCheckerboardPattern(float2 positionScaledClipSpace) {
-        const float colorLight = 1.0;
-        const float colorDark = 0.53;
+        float colorLight = materialParams.checkerboardGrays.x;
```

Comment:
I'm OK with this for now, however, I'd rather had these parameters constant folded into the shader code once tweaking is done. At least if I'm understanding it correctly, that the gradient curves are not needed to be changed after finetuning with Alex is done.

Reply:
I agree. Once we set out to do shader variations, we should do it for environments too. The checkerboard will need 2 variants for the public release: light and dark mode.